src/state/nftMarket/helpers.ts

```diff
@@ -27,3 +27,27 @@ export const getCollections = async (): Promise<any[]> => {
     return []
   }
 }
+
+export const getNftsFromCollection = async (collectionAddress: string): Promise<any[]> => {
+  try {
+    const res = await request(
+      GRAPH_API_NFTMARKET,
+      gql`
+        {
+          nfts(where: { collection: "${collectionAddress}" }) {
+            id
+            currentSeller
+            isTradable
+            tokenId
+            metadataUrl
+          }
+        }
+      `,
+    )
+
+    return res.nfts
+  } catch (error) {
+    console.error('Failed to fetch NFTs', error)
+    return []
+  }
+}
```

Comment:
We want to pass variables via the second parameter of `gql`. Check `state/predictions/helpers.ts` line 257 for a similar example.

Reply:
https://github.com/pancakeswap/pancake-frontend/blob/a9d7070c6975f1a868516ba6a9f36acbfc54945d/src/state/predictions/helpers.ts#L257

Reply:
You mean like here https://github.com/pancakeswap/pancake-frontend/blob/develop/src/state/info/queries/tokens/tokenData.ts#L72 ? 😄

Reply:
```
export const getBet = async (betId: string): Promise<BetResponse> => {
  const response = await request(
    GRAPH_API_PREDICTION,
    gql`
      query getBet($id: ID!) {
        bet(id: $id) {
          ...
          round {
            ...
          }
          user {
            ...
          } 
        }
      }
  `,
    {
      id: betId.toLowerCase(),
    },
  )
  return response.bet
}
```
This is the simplest example I can find. Notice `id: betId.toLowerCase(),` is passed in as the second parameter and the query is wrapped.